cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/util/BeanUtil.java

```diff
@@ -99,6 +98,20 @@ public BeanUtil(final String className)
     {
         this.className = className;
     }
+    
+    /**
+     * Get the base of the VOS URI for this running service.
+     * 
+     * @return The base VOS URI.
+     */
+    public static String getVosUriBase() {
+        String uriBase = System.getProperty(VOS_BASE_URI_PROP);
+        if (uriBase == null) {
+            // default to vault
+            return "vos://cadc.nrc.ca!vault";
```

Comment:
And in two places no less! ouch

What is this BeanUtil anyway? I've never seen it, no javadoc with an author, and it looks like a complicated way to do simple things. And the whole "bean" thing was a bad idea that died years ago :-)

Reply:
A very old class tied to the Restlet initialization sequence.  It will go when Restlet goes.  But I'll update it to use LocalServiceURI instead for now.